### ISSUE TITLE
Visually hidden content is not hidden from screen readers

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3164]
## Description  
Visually hidden content is not hidden from screen readers.  For example, when using VoiceOver/Safari/iOS 12.1.4 and swiping while the Hamburger Menu is open, you will hear content that is not visible due to the appearance of the Hamburger Menu. This forces screen reader users to hear information that was not intended for the user to read in the given application state. 

## Location     
clr-header

## Priority     
High

## Recommended fix  
Add aria-hidden="true" to the parent elements of any content that is visually obscured by the Hamburger Menu. 

## LOE  
Medium

**Comment by gnomeontherun** [comment]
This is related to our old website, closing as no longer valid.
